docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       - DOCKER_HOST=unix:///var/run/docker.sock
       - DEBUG=1
     volumes:
-      - "./tmp/localstack:/tmp/localstack"
+      - "./tmp/localstack:/var/lib/localstack"
```

Comment:
localstack doesn't support this directory anymore. The container crashed with error:

```
$ docker logs ff218dd6f98a
ERROR
============================================================================
  It seems you are mounting the LocalStack volume into /tmp/localstack.
  This will break the LocalStack container! Please update your volume mount
  destination to /var/lib/localstack.
  You can suppress this error by setting LEGACY_DIRECTORIES=1.

  See: https://github.com/localstack/localstack/issues/6398
============================================================================
```